File: Shared/Models/RouteStop.swift
import Foundation

// Parada del recorrido de un servicio.
struct RouteStop: Identifiable, Hashable {
    let stationId: Int
    let name: String
    let order: Int
    let scheduled: Date?
    let estimated: Date?
    let trackName: String?
    let hasPassed: Bool

    var id: Int { order }
}

File: android/app/src/main/java/com/alandaitch/anden/data/model/RouteStop.kt
package com.alandaitch.anden.data.model

import java.time.Instant

// Parada del recorrido de un servicio.
data class RouteStop(
    val stationId: Int,
    val name: String,
    val order: Int,
    val scheduled: Instant?,
    val estimated: Instant?,
    val trackName: String?,
    val hasPassed: Boolean
) {
    val id: Int get() = order
}
